yubico_piv_tool/yubico_piv_tool.mk

```diff
@@ -4,20 +4,18 @@
 #
 ################################################################################
 
-YUBICO_PIV_TOOL_VERSION = yubico-piv-tool-2.0.0
+YUBICO_PIV_TOOL_VERSION = yubico-piv-tool-2.2.0
 YUBICO_PIV_TOOL_SITE = git://github.com/Yubico/yubico-piv-tool.git
 YUBICO_PIV_TOOL_LICENSE = BSD
 YUBICO_PIV_TOOL_LICENSE_FILES = COPYING
 
-YUBICO_PIV_TOOL_CONF_ENV = HELP2MAN=true MAKEINFO=true
-YUBICO_PIV_TOOL_CONF_OPTS += --prefix=/usr
```

Comment:
We don't need to set the prefix here anymore?

Reply:
@JeremyBCD They changed their tooling from automake to cmake so most of the underlying build is different.